src/Templates/NewPowerShellScriptModule/editor/VSCode/tasks.json

```diff
@@ -1,42 +1,16 @@
-// Available variables which can be used inside of strings.
-// ${workspaceRoot}: the root folder of the team
-// ${file}: the current opened file
-// ${relativeFile}: the current opened file relative to workspaceRoot
-// ${fileBasename}: the current opened file's basename
-// ${fileDirname}: the current opened file's dirname
-// ${fileExtname}: the current opened file's extension
-// ${cwd}: the current working directory of the spawned process
 {
     // See https://go.microsoft.com/fwlink/?LinkId=733558
     // for the documentation about the tasks.json format
-	"version": "2.0.0",
-
-    // Start PowerShell
-    "windows": {
```

Comment:
I think we still need something like this to tell the OS to use PowerShell. Here's an example:
https://github.com/PowerShell/PowerShellEditorServices/blob/master/.vscode/tasks.json#L4-L24